pkgs/tools/security/opentype-sanitizer/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, freetype
+, gtest
+, lz4
+, meson
+, ninja
+, pkg-config
+, woff2
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
The modern approach nowadays is:

1. Use `pkgs/by-name/`.
2. Use `stdenv.mkDerivation(finalAttrs: {`...